README.md

```diff
@@ -123,6 +123,10 @@ A record of contract addresses used by Airseeker. If not specified, the addresse
 The address of the Api3ServerV1 contract. If not specified, the address is loaded from the Airnode protocol v1
 repository.
 
+###### DapiDataRegistry
```

Comment:
For now making it required, because it is not determined where the contract will be deployed (I guess it will stay in `dapi-management` repo).